server/migrations/20200405203608-create-store-info.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('StoreInfos', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      category: {
+        type: Sequelize.STRING
+      },
+      latitude: {
```

Comment:
can explore https://sequelize.org/master/class/lib/data-types.js~GEOMETRY.html for lat/lng, since they would be indexed by postGIS

Reply:
Add a field cooridinate